pandas/core/internals/array_manager.py

```diff
@@ -1103,11 +1103,16 @@ def _verify_integrity(self) -> None:
     def _normalize_axis(axis):
         return axis
 
-    def make_empty(self, axes=None) -> SingleArrayManager:
+    def make_empty(self, axes=None) -> Self:
         """Return an empty ArrayManager with index/array of length 0"""
         if axes is None:
             axes = [Index([], dtype=object)]
-        array: np.ndarray = np.array([], dtype=self.dtype)
+        #  error: No overload variant of "array" matches argument types
+        #  "list[<nothing>]", "ExtensionDtype"
+        array: np.ndarray = np.array(  # type: ignore[call-overload]
+            [], dtype=self.dtype
+        )
+        assert False  # is this even tested?
```

Comment:
Will remove it when the PR is ready - just want to test whether the CI reaches this (locally it wasn't reached).

Reply:
(self.dtype can also be a numpy dtype)